.github/workflows/szdiff.yml

```diff
@@ -0,0 +1,53 @@
+name: szdiff.yml
+on:
+  pull_request:
+
+# Cancel the workflow in progress in newer build is about to start.
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  cancel-in-progress: true
+
+jobs:
+  szdiff:
+    if: github.event.pull_request.draft == false
+    permissions:
+      contents: read
+      pull-requests: write
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code from pr
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.ref }}
+          repository: ${{ github.event.pull_request.head.repo.full_name }}
+          path: pr
+      - name: Checkout code from base
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.base.sha }}
+          path: base
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - name: Count Lines Of Code
+        run: |
+          pip install tabulate
+          BASE="$GITHUB_WORKSPACE/base"
+          PR="$GITHUB_WORKSPACE/pr"
+          cp "$PR/sz.py" .
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
```

Comment:
this seems kinda hacky but meh, can't we guarantee certain things won't appear in the output? cuz we are generating the output ourselves anyways

Reply:
The dd part was just following the [package's original example](https://github.com/marocchino/sticky-pull-request-comment#keep-more-than-one-comment) so I didn't dig too much.

Reply:
yea it's fine, just a small nit since we know our output and can guarantee that certain things won't appear in it.